pynicotine/pluginsystem.py

```diff
@@ -739,42 +735,53 @@ def plugin_settings(self, plugin_name, plugin):
         except KeyError:
             log.add_debug("No stored settings found for %s", plugin.human_name)
 
-    def get_command_descriptions(self, command_interface, search_query=None):
+    def get_command_list(self, command_interface):
+        """ Returns a list of every command and alias available. Currently used for
+        auto-completion in chats. """
 
-        command_groups = {}
+        commands = []
 
-        if command_interface == "chatroom":
-            command_list = self.chatroom_commands
+        for command, data in self.commands.get(command_interface).items():
+            commands.append(f"/{command} ")
```

Comment:
It causes a regression, somewhere an extra space creeps in if there are no aliases nor parameters, e.g:
`	/port   -  Check firewall state of user`
3 spaces ^^^ instead of 2.